### PR TITLE
[562] Migrate all static content linked in the footer

### PR DESCRIPTION
**Pull request by defong**
### Context
Footer
### Changes proposed in this pull request
Add the footer ad all the related ie, Accessibility Cookies Privacy policy Terms and conditions Service performance 

### Guidance to review

Ported across, like for like with minimum changes

https://teacher-training-api-pr-2446.london.cloudapps.digital/accessibility
https://teacher-training-api-pr-2446.london.cloudapps.digital/cookies
https://teacher-training-api-pr-2446.london.cloudapps.digital/guidance
https://teacher-training-api-pr-2446.london.cloudapps.digital/performance-dashboard
https://teacher-training-api-pr-2446.london.cloudapps.digital/privacy-policy
https://teacher-training-api-pr-2446.london.cloudapps.digital/terms-conditions
 
### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Migrate all static content linked in the footer](https://trello.com/c/0sszieWr/562-migrate-all-static-content-linked-in-the-footer)

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+    <%= tag.footer(class: "govuk-footer", role: "contentinfo") do %>
```
Comment (darokel) [inline app/views/layouts/_footer.html.erb:1]: Tab spacing is off here

**Review by darokel — APPROVED** [review]
Just one little comment but all good otherwise

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (defong) [reply app/views/layouts/_footer.html.erb:1]: fixed

**Review by mbocc — APPROVED** [review]
All good!

**Pull request merged**